themes/default/layouts/page/about.html

```diff
@@ -1,25 +1,173 @@
+                                    set of perspectives is essential to accomplishing our mission.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="lg:w-1/4 p-5">
+                    <div class="card bg-white max-w-md mx-auto">
+                        <div class="px-8 py-8">
+                            <div class="icon-section pb-4">
+                                <i class="fas fa-shoe-prints text-yellow-600 text-3xl"></i>
+                            </div>
+                            <div>
+                                <h6>Be Customer Driven</h6>
+                            </div>
+                            <div class="description">
+                                <p class="text-sm">
+                                    We work backwards from the customer, wearing their shoes
+                                    at all times. When our customers are successful, we are too.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="lg:w-1/4 p-5">
+                    <div class="card bg-white max-w-md mx-auto">
+                        <div class="px-8 py-8">
+                            <div class="icon-section pb-4">
+                                <i class="fas fa-balance-scale text-blue-600 text-3xl"></i>
+                            </div>
+                            <div>
+                                <h6>Lead with Openness</h6>
+                            </div>
+                            <div class="description">
+                                <p class="text-sm">
+                                    Information is open by default. We are transparent, open,
+                                    and honest with ourselves and with our customers.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="lg:flex ml-16 mr-16">
+                <div class="lg:w-1/4 p-5">
+                    <div class="card bg-white max-w-md mx-auto">
+                        <div class="px-8 py-8">
+                            <div class="icon-section pb-4">
+                                <i class="fas fa-running text-blue-600 text-3xl"></i>
+                            </div>
+                            <div>
+                                <h6>Bias for Action</h6>
+                            </div>
+                            <div class="description">
+                                <p class="text-sm">
+                                    We say yes more than we say no. Good today is better than
+                                    perfect tomorrow. When in doubt, experiment and fail fast.
```

Comment:
this is maybe a separate convo, but fail fast has a negative connotation now with some folks so we may want to figure out different phrasing. i do think this work though just ending on experiment though.